### DISCUSSION TITLE
Skiovox helper not able to be installed

### DISCUSSION THREAD

**Discussion by v4pl** [3kh0/ext-remover#1013] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
@AshtonDavies 
so i exited the skiovox for state testing but now i'm back in skiovox and my skiovox helper extension doesn't work. I'm checked my version and it's still 120.

**Comment by AshtonDavies** [comment]
Is the helper extension not installing, or is it not working?

**Reply by v4pl** [reply]
not installing like it says could not load manifest and blocked by admin

**Reply by v4pl** [reply]
i can't get in discord now so that's why i made a post here

**Reply by v4pl** [reply]
@AshtonDavies

**Reply by AshtonDavies** [reply]
Your administrator is setting policies within your kiosk applications, and yes, they can do that just like they can do for student profiles. Check another kiosk application if it has the same problem.

**Reply by v4pl** [reply]
like use another kiosk app?

**Reply by AshtonDavies** [reply]
> like use another kiosk app?

Yes

**Reply by v4pl** [reply]
Yep doesn't work

**Reply by AshtonDavies** [reply]
> Yep doesn't work

You have to move on from SKIOVOX then.

**Reply by v4pl** [reply]
I have different accounts should I try that?

**Reply by v4pl** [reply]
any other like webview exploits for 120?

**Reply by Blobby-Boi** [reply]
try using skiovox breakout maybe that works

**Reply by v4pl** [reply]
wdym? @Blobby-Boi

**Reply by Blobby-Boi** [reply]
use [skiovox breakout](https://github.com/MunyDev/skiovox-breakout)

**Reply by v4pl** [reply]
blocked by admin when i load the extension doesn't work

**Reply by v4pl** [reply]
i just need to add google account

**Reply by Blobby-Boi** [reply]
they can block extensions in skiovox?

**Reply by AshtonDavies** [reply, marked as answer]
> they can block extensions in skiovox?

They have control over the user profiles of the kiosk applications that they add to the Chromebooks. But I was a bit mistaken, because it's actually very limited, which makes more sense because SKIOVOX isnt supposed to be a thing.

All they can really do is restrict URLs, to prevent users from using webviews improperly.

[Look here for more information](https://support.google.com/chrome/a/answer/9273974?hl=en#zippy=%2Cstep-verify-policies-are-applied)

**Comment by v4pl** [comment]
Do u know the Google sign in link that works? cause that's all I really need

**Reply by Blobby-Boi** [reply]
chrome://chrome-signin/

**Reply by AshtonDavies** [reply]
https://tinyurl.com/AddSession
